package.json

```diff
@@ -10,15 +10,26 @@
     "lint": "eslint ./src/ --quiet"
   },
   "dependencies": {
+    "@felte/solid": "^1.2.3",
+    "@felte/validator-yup": "^1.0.9",
+    "@hope-ui/solid": "^0.6.7",
     "@solidjs/router": "^0.4.2",
+    "@stitches/core": "^1.2.8",
```

Comment:
Is this the theming library?

Reply:
@aburd Good catch.
I am not sure how it got it, but after confirming that Hope-UI is working without it and reinstalling hope-ui does not re-add it, I decided to remove it.